### PR TITLE
fix small sizing issue for options container

### PR DESCRIPTION
**Pull request by harisaurus**
Fixes https://github.com/Shopify/buy-button-js/issues/302

@tanema @tessalt @michelleyschen

### CODE REVIEW

**Comment by tessalt** [comment]
does this work in the adapter case too though?

**Comment by tessalt** [comment]
(lmk if you don't know how to find out)

File: src/styles/embeds/sass/components/_product.scss [hunk 1]
```diff
@@ -144,7 +145,7 @@
 }
 
 .shopify-buy__btn-wrapper {
-  margin-top: $gutter;
+  margin-top: $gutter + 5;
```
Comment (michelleyschen) [inline src/styles/embeds/sass/components/_product.scss]: why do we need a + 5 again?

**Comment by harisaurus** [comment]
@tessalt creating a branch to add the required fix for the adapter

File: src/styles/embeds/sass/components/_product.scss (hunk 1 above)
Comment (michelleyschen) [inline src/styles/embeds/sass/components/_product.scss]: that would be wonderful in the future!

**Pull request merged**